src/bench.h

```diff
@@ -125,6 +125,28 @@ int have_flag(int argc, char** argv, char *flag) {
     return 0;
 }
 
+/* check for invalid command line argument */
```

Comment:
You have a nice description of this function (including return values) in the commit message, but not here. Maybe move it here to a comment. It's not that helpful in the commit message.